tests/unit/test_queuehandler.py

```diff
@@ -16,7 +16,7 @@ def test_queue_handler_dispatch():
     assert record.levelno == picologging.DEBUG
     assert record.name == "test"
     assert record.msg == "test"
-    assert record.args == ()
+    assert record.args is None
```

Comment:
I confirmed that this is what happens with CPython logging. (It's because they explicitly set to None the unpicklable parts of the LogRecord).